Management_System/Management_System/Program.cs

```diff
@@ -25,4 +25,4 @@
 
 app.MapControllers();
 
-app.Run();
\ No newline at end of file
+await app.RunAsync();
```

Comment:
Why did you do this?

Reply:
I had reliability issue in sonar, that included those changes

Reply:
OK, but do you know what this for?